lib/browser.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import historyPolyfill from 'html5-history-api';
```

Comment:
If you need to support old browsers, you could also load this separately and put it in the global namespace. Many users don't have to support old browsers so this would add extra weight.